### PR TITLE
From Line

### PR DESCRIPTION
**Pull request by quizas211**
1. Unified variable names & brackets
2. Added some semi-trivial theorems in Line
3. Added a theorem that still needs to be proved in Ray
4. Added two theorems that still need to be proved in Colinear
5. My local files can be compiled successfully

### CODE REVIEW

File: EuclideanGeometry/Foundation/Axiom/Linear/Line.lean [hunk 1]
```diff
@@ -9,15 +9,22 @@ section setoid
 
 variable {P : Type _} [EuclideanPlane P]
 
-def same_extn_line : Ray P → Ray P → Prop := fun r₁ r₂ => r₁.toProj = r₂.toProj ∧ (r₂.source LiesOn r₁ ∨ r₂.source LiesOn r₁.reverse)
+def same_extn_line : Ray P → Ray P → Prop := fun r r' => r.toProj = r'.toProj ∧ (r'.source LiesOn r ∨ r'.source LiesOn r.reverse)
```
Comment (jjdishere) [inline EuclideanGeometry/Foundation/Axiom/Linear/Line.lean:12]: Thank you for your effort in making variables uniform！

File: EuclideanGeometry/Foundation/Axiom/Linear/Line.lean [hunk 2]
```diff
@@ -159,7 +167,28 @@ theorem ray_subset_line {r : Ray P} {l : Line P} (h : r.toLine = l) : r.carrier
...
+  apply Set.mem_of_subset_of_mem (ray_subset_line h) g
+
+theorem seg_subset_line {s : Seg_nd P} {l : Line P} (h : s.toLine = l) : s.1.carrier ⊆ l.carrier := by
+  intro A Ain
+  rw [← h]
+  apply seg_lies_on_Line Ain
+
+theorem pt_pt_lies_on_iff_seg_toLine {A B : P}{l : Line P}(h : B ≠ A) : A LiesOn l ∧ B LiesOn l  ↔ (Seg_nd.mk A B h).toLine = l := by 
+  constructor
+  · sorry
```
Comment (jjdishere) [inline EuclideanGeometry/Foundation/Axiom/Linear/Line.lean:182]: Still a sorry here to be filled

File: EuclideanGeometry/Foundation/Axiom/Linear/Line.lean [hunk 3]
```diff
@@ -194,7 +223,9 @@ theorem linear (l : Line P) {A B C : P} (h₁ : A LiesOn l) (h₂ : B LiesOn l)
       | inl c => sorry
       | inr c => sorry
 
-theorem maximal (l : Line P) {A B : P} (h₁ : A ∈ l.carrier) (h₂ : B ∈ l.carrier) (h : B ≠ A) : (∀ (C : P), colinear A B C → (C ∈ l.carrier)) := sorry
+theorem maximal' {l : Line P} {A B : P} (h₁ : A LiesOn l) (h₂ : B LiesOn l) (h : B ≠ A) : (∀ (C : P), colinear A B C → (C LiesOn l)) := by 
+  intro C Co
+  sorry
```
Comment (jjdishere) [inline EuclideanGeometry/Foundation/Axiom/Linear/Line.lean:228]: this theorem can be reforulate even better. we can move C，colinear A B C into hypothesis.

**Review by jjdishere — APPROVED** [review]

File: EuclideanGeometry/Foundation/Axiom/Linear/Line.lean (hunk 3 above)
Comment (quizas211) [reply EuclideanGeometry/Foundation/Axiom/Linear/Line.lean:228]: This theorem was written by otheres. So our team didn't make changes.

File: EuclideanGeometry/Foundation/Axiom/Linear/Line.lean (hunk 2 above)
Comment (quizas211) [reply EuclideanGeometry/Foundation/Axiom/Linear/Line.lean:182]: Yes, I was way too spleepy yesterday.

**Pull request merged**